schemas/base.cue

```diff
@@ -1,5 +1,7 @@
 {
-  displayed_name: string
-  kind: string
-  chart: _ // any type
-}
\ No newline at end of file
+	kind: string
+	display: {
+		name: string
```

Comment:
Just for you to know, like this the `display` struct is not closed, if you wanted it to be so you'd need to do something like
```
{
	kind: string
	display: #display
	options: _ // any type
}

#display: {
        name: string
}
```

but actually the wraping base def is already opened and should be in the current case:
```
#panel: {
        displayed_name: string
	kind: string
	chart: _ // any type
}

#panel
```


...but doing so was causing the validation of plugins to fail so I removed it "temporarily". Now I have again time to allocate to this, so you can keep it as is for now and I'll find a solution hopefully :D

Reply:
I'm not sure I'm following what's the issue here ^^. 

In the meantime, since we talking about `display`, for the moment there is only one attribute in this object, @LukeTillman @sjcobb do you foresee we will have more fields later ? Or do you think we could make it flat ? 

```cue
{
    kind: string
    display_name: string
    char: _
}
```

Reply:
It's not an issue in your PR, this was just to share information ;)

Reply:
alright let's keep it like that. Thanks!